android/app/src/main/java/com/on/turip/ui/common/model/PlaceModel.kt

```diff
@@ -1,186 +1,12 @@
-                    "3페이지 경포대 해수욕장",
-                    "관광지",
-                    """https://map.naver.com/p/search/%EA%
-                        |B2%BD%ED%8F%AC%EB%8C%80%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5/place/12079976?c=15.00,0,0,0,dh&placePath=/home?entry=bm
-                        |&from=map&fromPanelNum=2&timestamp=202507222010&locale=ko&svcName=map_pcv5&se
-                        |archText=%EA%B2%BD%ED%8F%AC%EB%8C%80
-                        |%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5
-                    """.trimMargin(),
-                ),
-                PlaceModel(
-                    "경포동해횟집",
-                    "음식점",
-                    """https://map.naver.com/p/search/%EA%
-                        |B2%BD%ED%8F%AC%EB%8C%80%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5/place/12079976?c=15.00,0,0,0,dh&placePath=/home?entry=bm
-                        |&from=map&fromPanelNum=2&timestamp=202507222010&locale=ko&svcName=map_pcv5&se
-                        |archText=%EA%B2%BD%ED%8F%AC%EB%8C%80
-                        |%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5
-                    """.trimMargin(),
-                ),
-                PlaceModel(
-                    "경포대 해수욕장3",
-                    "관광지",
-                    """https://map.naver.com/p/search/%EA%
-                        |B2%BD%ED%8F%AC%EB%8C%80%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5/place/12079976?c=15.00,0,0,0,dh&placePath=/home?entry=bm
-                        |&from=map&fromPanelNum=2&timestamp=202507222010&locale=ko&svcName=map_pcv5&se
-                        |archText=%EA%B2%BD%ED%8F%AC%EB%8C%80
-                        |%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5
-                    """.trimMargin(),
-                ),
-                PlaceModel(
-                    "경포대 해수욕장4",
-                    "관광지",
-                    """https://map.naver.com/p/search/%EA%
-                        |B2%BD%ED%8F%AC%EB%8C%80%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5/place/12079976?c=15.00,0,0,0,dh&placePath=/home?entry=bm
-                        |&from=map&fromPanelNum=2&timestamp=202507222010&locale=ko&svcName=map_pcv5&se
-                        |archText=%EA%B2%BD%ED%8F%AC%EB%8C%80
-                        |%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5
-                    """.trimMargin(),
-                ),
-                PlaceModel(
-                    "경포대 해수욕장5",
-                    "관광지",
-                    """https://map.naver.com/p/search/%EA%
-                        |B2%BD%ED%8F%AC%EB%8C%80%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5/place/12079976?c=15.00,0,0,0,dh&placePath=/home?entry=bm
-                        |&from=map&fromPanelNum=2&timestamp=202507222010&locale=ko&svcName=map_pcv5&se
-                        |archText=%EA%B2%BD%ED%8F%AC%EB%8C%80
-                        |%20%ED%95%B4%EC%88%98%EC%9A%95%EC%9E%A5
-                    """.trimMargin(),
-                ),
-            ),
-    )
+) {
+    fun placeUri(): Uri = mapLink.toUri()
```

Comment:
c: 우리 요런 건
```suggestion
    val placeUri: Uri
       get() = mapLink.toUri()
```
이렇게 쓰기로 했던 것 같아요!

매개변수 존재 -> 함수
매개변수 없음 -> 커스텀 getter

Reply:
반영 커밋 : [1331992](https://github.com/woowacourse-teams/2025-Turip/pull/79/commits/1331992e4ba40b436db5f2ec06d8776ec2eabe9d)